### ISSUE TITLE
confused by names - OUTPUT vs BACKUP variables

### ISSUE THREAD

**Issue by scr4bble** [rear/rear#920]
For me it was not very intuitive what does OUTPUT and BACKUP variables mean. I was confused a bit by these names when I was reading documentation for the first time. At start I didn't know why there are two destinations for backup and after reading documentation it sometimes confuses me which one is which.

* rear version (/usr/sbin/rear -V):
Relax-and-Recover 1.17.2 / Git

possible solutions:
Rename variable(s) (especially OUTPUT) to something more clear or explain it in the documentation somewhere near the start.

**Comment by gdha** [comment]
@scr4bble OUTPUT is for the rescue image location. BACKUP for backup/archive. Which piece of the documentation was not clear to you?

**Comment by jsmeix** [comment]
@scr4bble 
regarding "Rename variable(s)":

No, we cannot do that
without causing backward incompatible regressions
or without endlessly maintainig complicated mappings
between the old variable names and the new one
which also adds nothing else but confusion when
two different names have same meaning.

At least I will not actively maintain such overcomplications.

From my point of view the right solution is to
provide good documentation that makes it
obvious how things are meant.

Addendum:
The only reason why I would rename a variable is
when its old name is plain wrong which means
that its old name is totally misleading.

**Comment by scr4bble** [comment]
I think the main problem was that I didn't understand that data are treated separately from the image while making backup of the system. It sounds practical, but for me it was not clear at the start.
Maybe it could be explained in the description after this sentence: "Once that is done it initiates a restore from backup." ... something like "backup can be outside image", so user would know that by "backup" is not meant the image.
OUTPUT evokes in me output of backup tool - the backup itself... maybe if user understood this difference from description he wouldn't get so confused by names.

I wasn't the only one who gets confused so I wrote it here. Maybe others can share their point of view too. Or if this was not obvious only for me you can close this issue :)

**Comment by jsmeix** [comment]
I agree that it seems often (or only sometimes?)
users mix up the backup and restore of the files
with what Relax-and-Recover does because
I noticed questions on the "Relax-and-Recover users list"
that indicate it is not clear how Relax-and-Recover
versus backup and restore of the files are related.

For example see
http://lists.relax-and-recover.org/pipermail/rear-users/2016-June/003354.html
and
http://lists.relax-and-recover.org/pipermail/rear-users/2016-June/003366.html

Accordingly I think it should be more clearly
documented how Relax-and-Recover versus
backup and restore of the files are related.

**Comment by tcerna** [comment]
Hi, sorry for duplicate issue #933. I didn't know about this one.

I would like to remark that we had rear testday and most of participants told that names OUTPUT is confusing and we mixed terms OUTPUT and BACKUP.

**Comment by jsmeix** [comment]
Closing because "won't fix".
